packages/fxa-content-server/app/scripts/lib/router.js

```diff
@@ -48,6 +48,11 @@ import { getClientReactRouteGroups } from '../../../server/lib/routes/react-app/
 
 const NAVIGATE_AWAY_IN_MOBILE_DELAY_MS = 75;
 
+// React route groups specified here will effectively be set to
+// 100% roll out in production. Only add group names here once they've
+// been verified in production at the 15% experiment roll out.
+const ALWAYS_SHOWN_REACT_GROUPS = ['simpleRoutes'];
```

Comment:
I think this is fine, allows us to rollback from 100% if needed. We should probably file some issues around when we can completely remove the old route and only render the React version.

Reply:
@vbudhram I have mixed feelings on removing the old routes/code completely, because once we actually finish this project, we should be able to just remove the content-server package. We could spend the time now removing things especially to show old legacy code removed and for content-server to have a slightly smaller bundle along the way, or, we could do it all at once in the end.

Reply:
Yea, I guess I'm just thinking we have to maintain another list of converted routes here. I don't feel super strongly about it, but prefer the incremental deletes vs one big one at the end.